src/com/phonebookservice/exception/InternalServerException.java

```diff
@@ -0,0 +1,22 @@
+package com.phonebookservice.exception;
+
+/**
+ * Exception class internal server.
+ *
+ * @author Marwa Saleh
+ */
+public class InternalServerException extends HttpServiceException {
+    private static final int INTERNAL_STATUS_CODE = 500;
+    private static final String INTERNAL_ERROR_CODE = "Error found";
+    private static final String INTERNAL_ERROR_MESSAGE = "ERROR_FOUND";
+
+    /**
+     * Initialization of internal server exception.
+     *
+     * @param cause the case of exception.
+     */
+    public InternalServerException(final Throwable cause) {
+        super(INTERNAL_STATUS_CODE, INTERNAL_ERROR_MESSAGE, INTERNAL_ERROR_CODE,
```

Comment:
Do u see here, I can't know u did enter the string parameters in right order or not. It is very confusing. 
That's why we are using builder design pattern. 

We don't need it here but I am just saying FYI

Reply:
got it